bower.json

```diff
@@ -7,9 +7,15 @@
     "jquery.cookie": "~1.4.1",
     "bootstrap": "~3.3.1",
     "datatables-bootstrap3-plugin": "~0.2.0",
-    "angular": "~1.3.13",
+    "angular": "~1.3.15",
+    "angular-csv-import": "~0.0.15",
+    "angular-loading-bar": "~1.3.15",
     "angular-ui-router": "~0.2.13",
+    "angular-ui-select": "~0.11.2",
+    "angular-ui-tree": "~2.2.0",
     "angular-gettext": "~2.0.2",
+    "angular-sanitize": "~1.3.15",
+    "angular-xeditable": "~0.1.9",
```

Comment:
Have you checked the Licence of all the new dependencies? We can change the dependencies in the readme before the release, but if we can not use a dependency, we should know it now.

Also, are all this dependencies active developed?